compass/ocean/tests/hurricane/analysis/__init__.py

```diff
@@ -62,25 +64,25 @@ def setup(self):
         """
         package = self.__module__
 
-        if self.storm == 'sandy':
+        if self.storm == 'sandy' or self.storm == 'sandy_lts':
```

Comment:
The changes in this `if` would be reverted if you don't change `sandy` to `sandy_lts`.

Reply:
Hi Xylar, thanks for your feedback. If I leave `storm=sandy` regardless of whether lts is used, and pass that to Analysis, then wouldn't Analysis always pick up the non lts forward run? Now there is a `sandy_lts` test case folder where the forward output is saved when lts is on.